source/_integrations/nest.markdown

```diff
@@ -533,6 +533,8 @@ authentication process.
 
 - *Something went wrong: Please contact the developer of this app if the issue persists*: This typically means you are using the wrong type of credential (e.g. *Desktop Auth*). Make sure the credential in the [Google Cloud Console](https://console.developers.google.com/apis/credentials) is a *Web Application* credential following the instructions above.
 
+- *Something went wrong, please try again in few minutes.*: According to [Google Partner Connections Manager Error Reference](https://developers.google.com/nest/device-access/reference/errors/pcm) this error covers all other undocumented internal errors within Partner Connections. One of the issues that causes this error is the synchronisation problems between Nest App and Google Home App. Confirm that your Nest device is visible both on Nest App and Google Home under the same Home. If it is missing on Google Home, create a new dummy home on the Nest App as this triggers the synchronisation (the workaround as recommended by Google support team). Dummy entry can be deleted once the Nest Device is visible on Google Home.
```

Comment:
```suggestion
- *Something went wrong, please try again in few minutes.*: According to Google's [Partner Connections Manager Error Reference](https://developers.google.com/nest/device-access/reference/errors/pcm) this error covers all other undocumented internal errors within Partner Connections. One of the issues that causes this error is synchronisation problems between the Nest and Google Home apps. Confirm that your Nest device is visible within both apps under the same Home. If it is missing withinGoogle Home, create a new dummy home on the Nest App as this triggers the synchronisation (this is the workaround recommended by the Google support team). The dummy entry can be deleted once the Nest device is visible on with the Google Home app.
```
Slight reword suggestion

Reply:
If you are agreeing with that suggestion @cagriy, why don't you accept/merge it?

Reply:
Hi @frenck , on my end the PR still seems to be waiting for approval.

Reply:
No, there is a review comment above, including a suggestion.

As you haven't changed anything at this point, it is awaiting your action. Feel free to adjust the PR to the suggestion or commit the suggestion directly.

Marking this PR draft until is has been resolved.

../Frenck

Reply:
I am sorry, I reverted the draft by mistake, would you mind setting it as draft again?

Reply:
>  I reverted the draft by mistake, would you mind setting it as draft again?

But you made the change? You can draft it yourself if needed. Do you plan on making more changes?

Reply:
I think you are not aware of how the GitHub PR process works. You mark it ready when you think it is good for approval (that is now our decision to make). After that, a review will happen, by "someone". You have to await that.

Reply:
@cagriy You may find this [guide](https://developers.home-assistant.io/docs/review-process) handy, it's got a pretty good summary of the review process. If you have questions regarding the process after reading it/in the future the HA Discord is also a good place to ask :)